examples/widgets_minimal/cpp/qpushbutton.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QtWidgets/QPushButton>
+
+#include <memory>
+
+#include "rust/cxx.h"
+
+::std::unique_ptr<QPushButton>
+qpushbuttonNew();
```

Comment:
should we have a generic templated constructor in cxx-qt-lib to help with this?

Reply:
I think we have this now thanks to one of our other PRs that landed recently.
The QPainter one I think :thinking: